modAionImpl/src/org/aion/zero/impl/db/AionBlockStore.java

```diff
@@ -1178,12 +1159,12 @@ public void close() {
         lock.writeLock().lock();
 
         try {
-            indexDS.close();
+            index.close();
         } catch (Exception e) {
             LOG.error("Not able to close the index database:", e);
         } finally {
             try {
-                blocksDS.close();
+                blocks.close();
             } catch (Exception e) {
```

Comment:
does index.close() actually throw Exception?  it looks like it just throws IOException